### ISSUE TITLE
Can't start rTorrent

### ISSUE THREAD

**Issue by Meowmel** [Kerwood/rtorrent.auto.install#124]
Hi, just yesterday I had to reinstall my Raspberry Pi 2 (Raspian Wheezy) and thought that I would like to try your install script after my own rTorrent installation always had little problems (Linux beginner). 

The problem now: Everything went fine for the installation (Raspbian 3.0.2), but when I try to start rTorrent I get this:
![unbenannt2](https://cloud.githubusercontent.com/assets/10694804/10433191/51f02d00-7110-11e5-8257-7ace6c25de02.JPG)
First it asked me for a Password, I thought it would be my user (pi) password, that wasn't it. Second I tried my root password and got the Authentication error. Then I logged in as su to check if I had the right password, yup works. Tried the start command as root and again the "can't find rtorrent..." error.
What's going on, how can I fix this? Thanks for any help.

**Comment by Meowmel** [comment]
Yeah, rTorrent never got installed. Manually installed, pretty much everything else was fucked (ruTorrent etc.). Nuked that Raspian installation. 

Tried the Rtorrent-Auto-Install-1.1.0-Raspbian-Wheezy-Nginx Script on a fresh installation. 
![unbenannt4](https://cloud.githubusercontent.com/assets/10694804/10554358/1af9a6f4-7463-11e5-9449-c95354b853a5.JPG)
How about no.

**Comment by miracle091** [comment]
A [ is missed, add it at line 145

**Comment by Meowmel** [comment]
Added it to the file, worked. Now other problems occur:
![unbenannt5](https://cloud.githubusercontent.com/assets/10694804/10558824/706adcd6-74de-11e5-8315-d309fa065197.JPG)
After that it just continues to install rtorrent. Greeting me with the "Installation is complete" page. 
When accessing ruTorrent in my browser I get:
"Not Found
The requested URL /rutorrent was not found on this server.
Apache/2.2.22 (Debian) Server at 192.168.178.24 Port 80"

**Comment by miracle091** [comment]
Check if ```/var/www/rutorrent``` exist or change the temporany directorory at the beginning of the script and re-run it. Be sure the temp dir exist or have necessary space, the script need something like 20-30 MB.

**Comment by Meowmel** [comment]
WHAT THE FUCK MATE. That script just deleted all my data. 
Pointed the temp dir directly to my external HDD. Does the script just go rm * or what? 
![unbenannt3](https://cloud.githubusercontent.com/assets/10694804/10563370/28c97676-7588-11e5-9a32-4cbb54e42e8d.JPG)
Sadly it was already to late.

**Comment by miracle091** [comment]
Why don't you have created a new folder instead of use the root your hdd? WHY?!
Anyway finish install this command is executed:
```
if [ -d $TEMP_PLUGIN_DIR ]; then rm -rf $TEMP_PLUGIN_DIR; fi
```

**Comment by Meowmel** [comment]
Luckily my mistrust into the script let me do a backup yesterday evening, so nothing was lost. 
rm -rf $FUCK_EVERYTHING, are you by any chance else known as MrMEEE? *chuckles*

btw why I choose the root folder:
![unbenannt](https://cloud.githubusercontent.com/assets/10694804/10563803/29fd9b4c-7599-11e5-905c-0e1f08c1ad3e.JPG)
First I actually really tried the username again, that of course failed. Second I just chose a folder I knew should work and... yeah forgot to change it again.

**Comment by miracle091** [comment]
Try this version https://gist.github.com/miracle091/61aa929646bd5a3afed9
Is a one-script version instead of multiple ones, some folder (temp and temp-plugins) can be changed by run it.
The apache choise, works for me on debian and rpi.

**Comment by Meowmel** [comment]
* plays official Guinean pig anthem * 
![unbenannt4](https://cloud.githubusercontent.com/assets/10694804/10563996/e495670c-75a1-11e5-8597-3bf6e2ead04c.JPG)

btw5 the self created media/WD1TB/temp/ folder doesn't get deleted by your script (the nginx one) as well.

**Comment by miracle091** [comment]
Try now.
Is deleted at first run, finished installation and some specific point.

**Comment by Meowmel** [comment]
Still gives me
![unbenannt5](https://cloud.githubusercontent.com/assets/10694804/10564074/a803a57a-75a5-11e5-9b78-d071874d0bf4.JPG)
